config/components/kueue-viz/cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kueue-backend-read-access-binding
+  namespace: system
```

Comment:
This should be `kueue-system` by default I believe

Reply:
I fixed that. I was not doing proper namespace replacement in the template generation.
I updated the `update-helm.sh` script to handle that.
Now  if `--set enableKueueViz=true` is set, `kueue-viz` will be installed in the same `ns` as `kueue`

```
helm install kueue kueue/ --create-namespace --namespace kueue --set enableKueueViz=true
```

The refered subject for the cluster role binding is also updated.